packages/design-system/src/components/Checkbox/Checkbox.tsx

```diff
@@ -74,6 +74,7 @@ const CheckboxRoot = styled(Checkbox.Root)`
   align-items: center;
   // this ensures the checkbox is always a square even in flex-containers.
   flex: 0 0 2rem;
+  cursor: pointer;
```

Comment:
I think this style is too high up, it applies the pointer cursor to the whole area and not just to the checkbox:

![CleanShot 2024-09-27 at 12 38 46](https://github.com/user-attachments/assets/b410cf44-8f20-4abb-83dd-071de629520e)

Reply:
@remidej , I’ve implemented the changes,  and the hover effect now applies exclusively to the checkbox.

Reply:
@remidej Should the label be clickable, too? It prevents the user from being too precise

Reply:
Yes the label is clickable, but I don't think it should have the pointer cursor, so I like the way it looks in this PR

Reply:
LGTM 👍